packages/.vitepress/theme/components/FunctionInfo.vue

```diff
@@ -1,15 +1,19 @@
 <script setup lang="ts">
 import { useTimeAgo } from '@vueuse/core'
-import { computed } from 'vue'
+import { computed, ref } from 'vue'
 import { functions } from '@vueuse/metadata'
+import axios from 'axios'
 
 const props = defineProps<{ fn: string }>()
-
+const exportSizeMD = ref('')
 const info = computed(() => functions.find(i => i.name === props.fn))
 const format = (ts: number) => ago(-1, 'day')
 const lastUpdated = useTimeAgo(new Date(info.value?.lastUpdated || 0))
 const link = computed(() => `/functions\#category=${encodeURIComponent(info.value.category)}`)
 
+axios.get('/export-size.md').then(({ data }) => exportSizeMD.value = data)
```

Comment:
We probably don't want to be fetching this data on each page. Ideally it would work like the other function metadata and be fetched during build time. Maybe instead we can include function size inside of `@vueuse/metadata`

Reply:
@wheatjs Thank you for pointing that out, I placed it in metadata.ts and used fs.readFileSync.

Would you happen to know why the document is not rendering?
I can see that console logging it in metadata shows me the correct data that I am getting.

Reply:
I guess it's because code is being called in server and on client. There's no fs on client. 
How do i put a if(process.client) ?
process here doesn't have server or client prop

Reply:
I wanted to create a json file together with the export-size.md
but everytime i run pnpm size

<img width="1043" alt="image" src="https://user-images.githubusercontent.com/3810177/184429786-eadb8592-fa8f-4fc1-9b28-1ca92242591a.png">

I get this error.

Reply:
You should run build first and then analyse the size